chart/templates/worker-spawn-right-to-api.yaml

```diff
@@ -7,6 +7,9 @@ rules:
 - apiGroups: ["batch"]
   resources: ["jobs"]
   verbs: ["create"]
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims"]
```

Comment:
we will still need to handle the actual creation of this from the API code, right?